### PR TITLE
[Discussion] clang-format-10

### PR DESCRIPTION
**Pull request by rhaschke**
Adding to #2169, I tweaked the clang-format config even more.

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -40,12 +40,12 @@ ConstructorInitializerAllOnOneLineOrOnePerLine: true
...
 PointerBindsToType: true
 
-PenaltyExcessCharacter: 1000
-PenaltyBreakBeforeFirstCallParameter: 19
-PenaltyBreakComment: 60
-PenaltyBreakFirstLessLess: 1000
+PenaltyExcessCharacter: 50
+PenaltyBreakBeforeFirstCallParameter: 30
+PenaltyBreakComment: 1000
+PenaltyBreakFirstLessLess: 10
```
Comment (henningkayser) [inline .clang-format:46]: Interesting, what is the actual outcome of these changes? I didn't see too many changes with the STREAM macros for instance.

Comment (rhaschke) [reply .clang-format:46]: There are some examples, where a longer line is accepted, e.g. https://github.com/ros-planning/moveit/pull/2177/commits/e6a9b22035c607238dadf780c02120f32b6ba310#diff-3e42fc5ecfcb692fcc35b27df4d33675R171

**Comment by henningkayser** [comment]
> lgtm, will need to be rebased.

I pushed a rebased branch to https://github.com/ros-planning/moveit/pull/2169

**Comment by rhaschke** [comment]
As there was sufficient consent to merge this as early as possible, I'm going to merge this now.

**Pull request merged**